### DISCUSSION TITLE
Add Tree view

### DISCUSSION THREAD

**Discussion by allblue** [1Remote/1Remote#482] [Ideas]
Add a tree view in the display view, when there are more sessions, hierarchical management is more effective, tags cannot replace the hierarchical structure.

**Comment by majkinetor** [comment]
Why tags can not replace hierarchical structure? It's actually the opposite, tags are more flexible.

**Reply by allblue** [reply]
tags are indeed a flat way of presentation, and can not intuitively reflect the hierarchical relationship, if according to the “area-room-cabinet/function” logical relationship to organize, the tree structure is more clear and obvious. Secondly, the long-term maintenance of tags is also a problem, because tags may become outdated or inconsistent with the changes of time and users. Tags and tree structures have different adaptation scenarios and user groups, feeling can not completely replace.

**Reply by majkinetor** [reply]
>  tags may become outdated or inconsistent 

If there are a couple of tags instead of one representing the same thing, you can always rename the bad ones to keep the structure.  Also, nothing stops users inventing different categories of hierarchy for the same thing.

The other problems you mention are not convincing either, I think you just gravitate toward hierarchical categories because that is what you used so far.
